modules/lsp/src/main/scala/playground/lsp/Main.scala

```diff
@@ -16,44 +16,23 @@ import java.io.InputStream
 import java.io.OutputStream
 import java.io.PrintStream
 import java.io.PrintWriter
-import java.nio.charset.Charset
 
 object Main extends IOApp.Simple {
-
   private val logOut = new PrintStream(new FileOutputStream(new File("smithyql-log.txt")))
   private val logWriter = new PrintWriter(logOut)
-
-  implicit val ioConsole: std.Console[IO] =
-    new std.Console[IO] {
-
-      def readLineWithCharset(
-        charset: Charset
-      ): IO[String] = IO.consoleForIO.readLineWithCharset(charset)
-
-      def print[A](a: A)(implicit S: Show[A]): IO[Unit] = IO(logWriter.print(a.show))
-
-      def println[A](a: A)(implicit S: Show[A]): IO[Unit] = IO(logWriter.println(a.show))
-
-      def error[A](a: A)(implicit S: Show[A]): IO[Unit] = IO(logWriter.print("ERROR: " + a.show))
-
-      def errorln[A](a: A)(implicit S: Show[A]): IO[Unit] = IO(
-        logWriter.println("ERROR: " + a.show)
-      )
-
-    }
-
-  def log[F[_]: std.Console](s: String): F[Unit] = std.Console[F].println(s)
-
-  def run: IO[Unit] = {
-    val stdin = System.in
-    val stdout = System.out
-
-    IO(System.setOut(logOut)).toResource *>
-      launch(stdin, stdout)
-  }
-    .use { launcher =>
-      IO.interruptibleMany(launcher.startListening().get())
-    } *> log("Server terminated without errors")
+  private val stdin_raw = System.in
+  private val stdout_raw = System.out
+
+  def run: IO[Unit] =
+    (
+      IO(System.setOut(logOut)).toResource *>
```

Comment:
This ensures we override the stdout stream for everything early on (or... at least it _should_ be doing that).

That should allow us to use the default `Console[IO]` insteande to log to `smithyql-log.txt`.